src/css/wizard-navigation-bar.scss

```diff
@@ -13,8 +13,8 @@
 // indicator state: 'default' or 'hover'
 $wz-param-indicator-state: null !global;
 
-// base color for one of possible wizard step states (default, current, done etc.).
-$wz-param-indicator-color: null !global;
+// wizard step state: 'default', 'current', 'done' etc.
+$wz-param-step-state: null !global;
 
 
 @mixin wz-horizontal-line($dot-width, $dot-height, $line-color) {
```

Comment:
Maybe we should add some comments to the mixins which contain what the mixins do

Reply:
I think you missed this comment

Reply:
Yes, missed it, thanks! Comments added in 32b3c1e